dot/state/grandpa_test.go

```diff
@@ -126,7 +127,7 @@ func TestGrandpaState_LatestRound(t *testing.T) {
 func testBlockState(t *testing.T, db *chaindb.BadgerDB) *BlockState {
 	ctrl := gomock.NewController(t)
 	telemetryMock := NewMockTelemetry(ctrl)
-	telemetryMock.EXPECT().SendMessage(gomock.Any()).AnyTimes()
+	telemetryMock.EXPECT().SendMessage(gomock.Any()).Times(1)
```

Comment:
do we not know what type of telemetry message  that gets sent here?

Reply:
We can know the types but not the value data, It's a helper function to use in other tests and I have to do some refactor to pass this params in the different tests that use this function.
I think it's out of scope for this PR, what do you think?

Reply:
you can use `gomock.AssignableToTypeOf()`.

`telemetryMock.EXPECT().SendMessage(gomock.AssignableToTypeOf(StructName{})).Times(1)`

Reply:
Thanks!